staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go

```diff
@@ -56,7 +56,12 @@ func WithAudit(handler http.Handler, sink audit.Sink, policy policy.Checker, lon
 		}
 
 		ev.Stage = auditinternal.StageRequestReceived
-		processAuditEvent(sink, ev, omitStages)
+		if !processAuditEvent(sink, ev, omitStages) {
+			audit.ApiserverAuditDroppedCounter.Inc()
+			utilruntime.HandleError(fmt.Errorf("failed to store audit event"))
```

Comment:
does this print a multi-line stack trace? Do we really want that?

Reply:
Right, having the metric is enough, logging a static text with stack trace doesn't bring any additional value. Removing.